app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
```

Comment:
The line continuation character (\) is not recommended for breaking long lines. Instead, you can use parentheses to make the code more readable and to avoid W504 errors.